packages/lms/src/routes/+layout.svelte

```diff
@@ -14,7 +14,9 @@
   import type { LayoutData } from './$types';
   import EntityNav from '$lib/components/navigation/EntityNav.svelte';
   import { page } from '$app/stores';
-
+  import hljs from '$lib/utils/highlightjs.config';
+  ('../lib/utils/highlightjs.config');
```

Comment:
What does this line do?

Reply:
Skeleton UI´s CodeBlock component uses Highlight js instead of prismjs. Using Skeleton overrules prismjs apparently. this line is just configuring highlightJS for syntax highlighting in codeblocks.